### ISSUE TITLE
[Bug] Behaviour changed for middle-click window title in gnome 46

### ISSUE THREAD

**Issue by lonix1** [electron/electron#44528]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

30.5.1 (the version used in the latest vscode)

### What operating system(s) are you using?

Ubuntu

### Operating System Version

ubuntu 24.04

### What arch are you using?

x64

### Expected Behavior

I use the mouse middle click (on a window's titlebar) to lower the window - so I can easily cycle through open windows. This setting is set via the shell, or the gnome tweaks tool.

### Actual Behavior

That changed in when upgrading to gnome 46. It doesn't honor that gnome setting anymore. So a middle click on the titlebar does nothing.

I've experienced this bug in a number of electron-based apps, including vscode.

**Comment by dsanders11** [comment]
> This setting is set via the shell, or the gnome tweaks tool.

Could you please provide the exact setting for repro purposes?

> That changed in when upgrading to gnome 46. 

What was the previous GNOME version?

**Comment by ak2766** [comment]
Confirmed in Gnome: https://gitlab.gnome.org/GNOME/mutter/-/issues/2778

**Comment by ak2766** [comment]
bump

**Comment by ak2766** [comment]
@dsanders11 - FYI!

TL;DR: I think this is more a Gnome issue for me so not sure if I'm helping by adding my noise here.  Let me know.

For my case, here's what it takes to replicate this issue.

* Clean install of Ubuntu 24.04
* Ensure that `WaylandEnable=false` is NOT commented out in `/etc/gdm3/customer.conf`
* Reboot
Once I reboot, I do not see the option to select either X11 or Wayland.  However, I have this:
```
$ > loginctl
SESSION  UID USER SEAT  TTY  STATE  IDLE SINCE
      4 1000 akk  seat0 tty2 active no   -    

1 sessions listed.
$ > loginctl show-session 4 -p Type
Type=x11
```

For now, I note that the following apps are obeying the middle-click on title bar:
* Files
* Kate
* Sublime Text
* Vysor

I'm going to test this again in a clean install of 24.04.2 VM and update here if I find I missed something.  I've upgraded my laptop from a clean install of 24.04.1 to 24.04.2 but the behavior hasn't changed.

**Comment by codebytere** [comment]
this turned out to be mutter (https://gitlab.gnome.org/GNOME/mutter/-/issues/2778) - with server-side decorations the titlebar click never reaches us. closing since there's nothing to change in electron.
